File: leetsync/2107-find-unique-binary-string/find-unique-binary-string.java
class Solution {
    public String findDifferentBinaryString(String[] nums) {
        String answer = "";
        for (int i = 0; i < nums.length; i++) {
            char ch = nums[i].charAt(i);
            if (ch == '1') {
                answer += '0';
            } else {
                answer += '1';
            }
        }
        return answer;
    }
}

File: leetsync/2107-find-unique-binary-string/find-unique-binary-string.go
func findDifferentBinaryString(nums []string) string {
	answer := make([]byte, len(nums[0]))
	for index, value := range nums {
		ch := value[index]
		if ch == '0' {
			answer[index] = '1'
		} else {
			answer[index] = '0'
		}
	}
	return string(answer)
}